go.mod

```diff
@@ -176,3 +176,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/mongodb/mongodb-atlas-kubernetes/v2 v2.4.1 => ../mongodb-atlas-kubernetes
```

Comment:
This is required for local development, but will not work on the CI, obviously.

This PR needs to wait for the 2.5 release to happen, or at least for the `v2.5.0` to exist.

Reply:
Removed, ready to go now